apps/wallet/src/ui/app/shared/Dialog.tsx

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
a decent bit of this is inspired by / copy + pasted from Jordan's implementation at https://github.com/MystenLabs/sui/blob/2104fb8514a26553c49010a392d8dff59249cd57/dapps/multisig-toolkit/src/components/ui/dialog.tsx